test/feedcontainer.cpp

```diff
@@ -382,6 +382,60 @@ TEST_CASE("sort_feeds() sorts by number of articles in a feed "
 	}
 }
 
+TEST_CASE("sort_feeds() and keep in-group order", "[FeedContainer]")
```

Comment:
Can this test please be extended to other sorting fields?

Reply:
do you mean a test for each feed sort method?

Reply:
Yes. Since they're implemented separately, we got to have a test for each. If we don't do this now, then sometime in the future someone might change something in one of the methods, see that tests pass, and conclude that everything's fine, even though they might've broken the results of this PR.